inbm/dispatcher-agent/dispatcher/fota/bios_factory.py

```diff
@@ -50,7 +50,6 @@ def extract_ext(file_name: str) -> Optional[str]:
     @param file_name: name of the file that is downloaded
     @return: returns as a package or a cert otherwise None
     """
-    logger.debug("inside extract")
     logger.debug(f"file_name={file_name}")
```

Comment:
**<h2>$\color{#FFD700}Coding Style$</h2>**
**Suggestion:** Ensure consistent use of single quotes for strings to maintain code consistency.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    logger.debug('file_name={}'.format(file_name))
```